src/css/typography.sass

```diff
@@ -0,0 +1,165 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans&family=Poppins:wght@400;500;600;700&display=swap")
+@import 'colors'
+
+// $thin: 100
+// $extra-light: 200
+// $light: 300
+$regular: 400
+$medium: 500
+$semi-bold: 600
+$bold: 700
+// $extra-bold: 800
+// $black: 900
+// $extra-black: 950
```

Comment:
uncomment only the "font-weights" that are needed for the typography of the project. That way we can see what weights this project uses and we can make sure the google fonts URL imports those weights.